src/proxies/url_state.js

```diff
@@ -57,4 +57,13 @@ UrlState.load = function() {
   });
 };
 
+UrlState.getShardValue = function(shardPrefix) {
```

Comment:
This is weird: from the name of this function, we'd expect it to return a value or it returns a boolean. Is it the expected behavior? If so, please rename the function.

Reply:
Indeed, thanks for info. TIL.